src/app/RootRoute.js

```diff
@@ -98,8 +98,6 @@ export default {
             //require.ensure([], (require) => {
             cb(null, [require('app/components/pages/PostPage')]);
```

Comment:
is there an advantage to doing the inline require here, rather than importing the component at the top like we're doing with `PostsIndex`?

Reply:
Yes, it gives us an option to use webpack's dynamic loading, as it's already done for the market page - all it requires is to uncomment require.ensure block.